src/main/resources/native/android/c/launcher.c

```diff
@@ -46,6 +46,7 @@ char *appDataDir;
 char *timeZone;
 
 int start_logger(const char *app_name);
+int JNI_OnLoad_sunec(JavaVM *vm, void *reserved);
```

Comment:
Add a comment that links to the issue in GraalVM, so that we can safely revert this commit once it is fixed in a release that we support?

Reply:
Done